nflow-engine/src/main/resources/scripts/db/update-7.1.0-x/postgresql.update.ddl.sql

```diff
@@ -0,0 +1,25 @@
+drop index idx_workflow_parent;
+create index concurrently idx_workflow_parent on nflow_workflow(parent_workflow_id) where parent_workflow_id is not null;
+
+drop index idx_workflow_archive_parent;
+create index concurrently idx_workflow_archive_parent on nflow_archive_workflow(parent_workflow_id) where parent_workflow_id is not null;
+
+alter table nflow_workflow set (fillfactor=95);
+
+alter index nflow_workflow_action_pkey set (fillfactor=100);
+alter index nflow_workflow_action_workflow set (fillfactor=100);
+alter index pk_workflow_state set (fillfactor=100);
+
+drop index nflow_archive_workflow_uniq;
```

Comment:
Why this is removed?

Reply:
The uniqueness is no longer important in archive. It is meant for the to support idempotent insert of workflow instances.
So I was thinking that archive might actually benefit more from index on type, which I added instead.

Reply:
But maybe I should drop that archive index from all databases?

Reply:
I'm not sure, maybe it's good it's dropped, then you can add new instance to prod with same extid and type, and archive that even though there is already such item in archive. It might be good idea to document this somewhere where we say that type+extid is unique. At least highlight this is release notes.
And yes, if it's dropped, it should be dropped from all databases.

Reply:
Let's drop this from all databases.